packages/web/components/RegisterPlayer.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Button,
+  ButtonGroup,
+  EmailIcon,
+  Icon3box,
+  MetaHeading,
+} from '@metafam/ds';
+import React from 'react';
+
+import { FlexContainer } from './Container';
+
+export const RegisterPlayer: React.FC = () => {
+  return (
+    <FlexContainer flex={1}>
+      <MetaHeading m={5}>Register your Player account</MetaHeading>
+      <ButtonGroup spacing={5} mt={20}>
+        <RegisterButton textTransform="uppercase">
+          <Icon3box mr={2} /> box
+        </RegisterButton>
+        <RegisterButton>GitHub</RegisterButton>
+        <RegisterButton>
+          <EmailIcon mr={2} />
+          Email
+        </RegisterButton>
+      </ButtonGroup>
+    </FlexContainer>
+  );
+};
+
+const RegisterButton: React.FC<typeof Button> = ({ children, ...props }) => (
```

Comment:
To get the correct prop types, you will have to do:
```suggestion
const RegisterButton: React.FC<React.ComponentProps<typeof Button>> = ({ children, ...props }) => (
```

But, easier way to do it is to import/export the `ButtonProps` type from chakra

Reply:
In `chakra-ui@next` they are not exporting `ButtonProps` or `LinkProps`, and I get the following error when using them:
```
components/RegisterPlayer.tsx(31,32): error TS2709: Cannot use namespace 'ButtonProps' as a type.
```

Reply:
So I'm using `React.ComponentProps<typeof Button>` itself.